app/assets/stylesheets/variables.css

```diff
@@ -1,3 +1,14 @@
 :root {
   --stanford-illuminating-dark: #fec51d1a;
+  --stanford-cardinal-dark: #820000;
 }
+
+.stanford-digital-red-link {
```

Comment:
I don't think we want these styles in `variables.css`, do we? How about `component-library-overrides.css` instead?

Reply:
Is it really a component override though? Maybe because it is overriding a link? I originally thought it was kind of a variable because if we use the stanford-digital-red-link elsewhere isn't it kind of a variable. Like how we do bg-blue in bootstrap. Honestly I had a lot of trouble figuring out where this went.